### ISSUE TITLE
Cannot DROP or TERMINATE when names of streams, tables, queries are quoted (backticks)

### ISSUE THREAD

**Issue by tkaven** [confluentinc/ksql#2171]
```show streams;

Stream Name                 | Kafka Topic                 | Format 
--------------------------------------------------------------------
ks.hgame.projects.11_stream | ks.hgame.projects.11_stream | JSON   
vs.hgame.projects.1_stream  | maxwell.hgame.projects      | JSON   
vs.hgame.projects.n1_stream | maxwell.hgame.projects      | JSON   
--------------------------------------------------------------------
ksql> drop stream `vs.hgame.projects.1_stream`;
Cannot drop vs.hgame.projects.1_stream. 
The following queries read from this source: [CSAS_ks.hgame.projects.11_stream_3]. 
The following queries write into this source: []. 
You need to terminate them before dropping vs.hgame.projects.1_stream.
ksql> terminate `CSAS_ks.hgame.projects.11_stream_3`;
java.lang.Exception: No running query with id `CSAS_ks.hgame.projects.11_stream_3` was found
ksql> show queries;

Query ID                           | Kafka Topic                 | Query String                                                                                                                                                                                                                                                                                                                                                                                                                                                 
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 CSAS_ks.hgame.projects.11_stream_3 | ks.hgame.projects.11_stream | CREATE STREAM `ks.hgame.projects.11_stream` WITH (VALUE_FORMAT='JSON') AS SELECT data->`userid` AS userid, data->`iscancel` AS iscancel, data->`totalprice` AS totalprice, data->`bouns` AS bouns, data->`bonustime` AS bonustime, data->`writetime` AS writetime, data->`deducttime` AS deducttime, data->`projectid` AS projectid, data->`lotteryid` AS lotteryid, data->`methodid` AS methodid, data->`modes` AS modes FROM `vs.hgame.projects.1_stream`; 
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
For detailed information on a Query run: EXPLAIN <Query ID>;
ksql> terminate `CSAS_ks.hgame.projects.11_stream_3`;
java.lang.Exception: No running query with id `CSAS_ks.hgame.projects.11_stream_3` was found
ksql> 
```

**Comment by tkaven** [comment]
CSAS_ks.hgame.projects.11_stream_3

cause 11 not alowd

**Comment by miguno** [comment]
@tkaven : Which version of KSQL are you using?

**Comment by tkaven** [comment]
> @tkaven : Which version of KSQL are you using?

version is 5.0.1

thank you

**Comment by miguno** [comment]
@big-andy-coates : I suppose this is, again, related to quoted identifiers (i.e., the use of backticks)?

**Comment by apurvam** [comment]
Seems like a dup of #1888

**Comment by agavra** [comment]
This issue should be fixed as of master!
